### DISCUSSION TITLE
Webhook for specific domain

### DISCUSSION THREAD

**Discussion by johndo100** [TimothyYe/godns#253] [Q&A]
Hi, thanks for your hard work.
I'm using godns for quite a long time and it work well.
Recently I need Webhook to run a bash script every time ip change.
I tried but Webhook sent multi messages to my endpoint for every domain.
How can I make it work for specific domain and multi endpoint.
Please help.

**Comment by TimothyYe** [comment]
So, would you like to send the webhook messages to the specific domain and multiple endpoints?

**Reply by johndo100** [reply]
Yes.
For example I have several domains:
- domain1.com
- domain2.com
- domain3.com
And several webhook endpoints:
- http://endpoint1.com:9000/hooks/godns
- http://endpoint2.com:9000/hooks/godns

I want to send only one webhook message for domain1 to endpoint1.
One webhook message for domain2 to endpoint2.
Now I only have one endpoint and godns send multiple messages to this endpoint which include all domains I have.
